Does not compile: @compileError("deprecated; use @tagName or @errorName directly")

### Zig Version

0.11.0-dev.3058+cd7e2bf57

### Zig Language Server Version

d3f127e78eb7c881768c3a7701b0af8e4c8315a7

### Steps to Reproduce

`zig build -Doptimize=ReleaseSafe` (also just `zig build`)

### Expected Behavior

Build `zls`

### Actual Behavior


Tried clearing local and global caches to no avail. Intel mac Ventura 13.3.

```zig build-exe zls ReleaseSafe native: error: the following command failed with 1 compilation errors:
/usr/local/Cellar/zig/HEAD-cd7e2bf/bin/zig build-exe /zls-path/zls/src/main.zig -OReleaseSafe --cache-dir /zls-path/zls/zig-cache --global-cache-dir /home/.cache/zig --name zls --mod known-folders::/home/.cache/zig/p/122028c00915d9b37296059be8a3883c718dbb5bd174350caedf152fed1f46f99607/known-folders.zig --mod tres::/home/.cache/zig/p/1220beaae8d152baa941a10b7ef3d3a59d093b257047035e2373c3c2f876ad29ccc8/tres.zig --mod build_options::/zls-path/zls/zig-cache/c/1c1dcd8af49cd107732cd5a01108114b/options.zig --mod diffz::/home/.cache/zig/p/1220ed4aed884221108ad39f2658b69a91653e0bbc8ce429bc7f1bc4e58f6a751553/DiffMatchPatch.zig --deps build_options,known-folders,tres,diffz --listen=- 
Build Summary: 1/4 steps succeeded; 1 failed (disable with -fno-summary)
install transitive failure
└─ install zls transitive failure
   └─ zig build-exe zls ReleaseSafe native 1 errors
      └─ options cached
/usr/local/Cellar/zig/HEAD-cd7e2bf/lib/zig/std/meta.zig:17:21: error: deprecated; use @tagName or @errorName directly
pub const tagName = @compileError("deprecated; use @tagName or @errorName directly");
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    initializeHandler: src/Server.zig:463:54
    processMessage: src/Server.zig:1460:26
    remaining reference traces hidden; use '-freference-trace' to see all reference traces```

--- a/src/Server.zig
+++ b/src/Server.zig
@@ -460,7 +460,7 @@ fn initializeHandler(server: *Server, request: types.InitializeParams) Error!typ
 
     log.info("zls initializing", .{});
     log.info("{}", .{server.client_capabilities});
-    log.info("Using offset encoding: {s}", .{std.meta.tagName(server.offset_encoding)});
+    log.info("Using offset encoding: {s}", .{@tagName(server.offset_encoding)});
 
     server.status = .initializing;
 

